pkg/runner/cluster_k8s.go

```diff
@@ -187,10 +187,7 @@ func (*ClusterK8sRunner) Run(ctx context.Context, input *api.RunInput, ow io.Wri
 				if cfg.KeepService {
 					return
 				}
-				client, err := pool.Acquire(ctx)
-				if err != nil {
-					log.Errorw("couldn't get client from pool", "pod", podName, "err", err)
-				}
+				client := pool.Acquire()
```

Comment:
If the main context gets canceled, I still want to get a client and do clean-up. The context in the pool is generally not needed.